content/guides/outliers.md

```diff
@@ -19,7 +19,7 @@ Outlier Detection is an algorithmic feature that allows you to detect when some
 
 ## How to Use Outlier Detection on Your Data
 
-We’ve added a new query function called `outliers` to our query language. This function will return the usual results but outlier series will be marked.
+We’ve added a query function called `outliers` to our query language. This function will return the usual results but outlier series will be marked.
```

Comment:
Good point. Changing to  'The `outliers` query function, when applied to your query, will return the usual results but with outlier series marked.'